src/transformers/models/bert/modeling_tf_bert.py

```diff
@@ -80,8 +79,6 @@
 _CHECKPOINT_FOR_QA = "ydshieh/bert-base-cased-squad2"
 _QA_EXPECTED_OUTPUT = "'a nice puppet'"
 _QA_EXPECTED_LOSS = 7.41
-_QA_TARGET_START_INDEX = 14
```

Comment:
Currently these are not used, so I understand the reason you remove them.

But we should instead keep here and use them in `add_code_sample_docstrings` for `BertForQuestionAnswering`.

(without using it, it works currently due to the default values `14` and `15` specified in `def add_code_sample_docstrings` - this is not the best approach though)